addons/frag/functions/fnc_dev_clearTraces.sqf

```diff
@@ -0,0 +1,24 @@
+#include "..\script_component.hpp"
+/*
+ * Author: Lambda.Tiger
+ * Cleares all dev spheres and traces
+ *
+ * Arguments:
+ * None
+ *
+ * Return Value:
+ * None
+ *
+ * Example:
+ * [] call ace_frag_fnc_dev_clearTraces;
+ *
+ * Public: No
+ */
+GVAR(dev_trackLines) = createHashMap;
+for "_i" from 0 to count GVAR(dev_eventSpheres) - 1 do
+{
+    deleteVehicle (GVAR(dev_eventSpheres)#_i);
+};
+GVAR(dev_eventSpheres) = +[];
```

Comment:
I'm curious if `+` (return a new copy of) does anything here, or if it was a typo.

Reply:
It is not needed, I think at one point there was a point to it, but as mentioned here https://github.com/acemod/ACE3/pull/9728#discussion_r1452649777, I updated some old debug code and spiffed it up, but apparently not all the way.